modules/ocf_keyserver/templates/sks-web-nginx.erb

```diff
@@ -0,0 +1,45 @@
+server {
+
+    listen 80 default_server;
+    listen [::]:80 default_server;
+    listen 443 default_server ssl;
+    listen [::]:443 default_server ssl;
+    listen 11371;
+    listen [::]:11371;
+
+    ssl_certificate /etc/ssl/private/<%= @fqdn %>.bundle;
+    ssl_certificate_key /etc/ssl/private/<%= @fqdn %>.key;
+    ssl_dhparam /etc/ssl/dhparam.pem;
+    ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
+
+    server_name _;
+    server_name <%= @fqdn %>;
+    server_name <%= @hostname %>;
+    server_name *.sks-keyservers.net;
+    server_name *.pool.sks-keyservers.net;
+    server_name pgp.mit.edu;
+    server_name keys.gnupg.net;
+
+    root /var/www/html;
+
+    rewrite ^/stats /pks/lookup?op=stats;
+    rewrite ^/s/(.*) /pks/lookup?search=$1;
+    rewrite ^/search/(.*) /pks/lookup?search=$1;
+    rewrite ^/g/(.*) /pks/lookup?op=get&search=$1;
+    rewrite ^/get/(.*) /pks/lookup?op=get&search=$1;
+    rewrite ^/d/(.*) /pks/lookup?op=get&options=mr&search=$1;
+    rewrite ^/download/(.*) /pks/lookup?op=get&options=mr&search=$1;
```

Comment:
Why are these needed? Why not use the URLs provided by the webapp?

Reply:
the web application itself requires these URLs